src/java.base/share/classes/java/security/cert/X509Certificate.java

```diff
@@ -599,6 +608,9 @@ public X500Principal getSubjectX500Principal() {
      * and it provides a default implementation. Subclasses
      * should override this method with a correct implementation.
      *
+     * @implNote The JDK SUN provider supports the third and fourth
+     * otherName elements.
```

Comment:
I would probably use "entries" instead of "elements" to be consistent with the previous wording.

Reply:
Good.